Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,33 @@
 import React from 'react';
-import './Button.css';
+import './Button.scss';
+
+const ActionButton = props => {
+	console.log(props)
+
+	return (
+		<button className={`${props.actionStyles}`}>{props.text}</button>
+	)
+}
+
+// ActionButton.defaultProps = {
+// 	actionStyles: 'action',	
+// }
+
+const Action = () => {
+
+	ActionButton.defaultProps = {
+		actionStyles: 'action',	
+	}
+
+	return (
+		<div className="action-buttons">
+			<ActionButton text="&divide;"/>
+			<ActionButton text="&times;"/>
+			<ActionButton text="&#8722;"/>
+			<ActionButton text="&#43;"/>
+			<ActionButton text="&#61;"/>
+		</div>
+	)
+}
+
+export default Action
```

Comment:
this is a slightly awkward and confusing way to get these buttons rendered. in general you want to default your export to the same component that the filename references. there is probably a less tricky way to achieve the proper visual display you are getting.